src/main/java/io/lettuce/core/pubsub/RedisPubSubListener.java

```diff
@@ -77,4 +77,14 @@ public interface RedisPubSubListener<K, V> {
      */
     void punsubscribed(K pattern, long count);
 
+    /**
+     * Subscribed to a Shard channel.
+     *
+     * @param shardChannel Shard channel
+     * @param count Subscription count.
+     */
+    default void ssubscribed(K chashardChannelnnel, long count) {
+        throw new UnsupportedOperationException(
+                "This operation is not available by default. Use one of available pub/sub adapters or override this method in your class!");
+    }
```

Comment:
Have you considered delegating the default implementation to `subscribed` for easier adoption?

Reply:
honestly, i did not.
i am trying to anticipate cases where this would be unexpected behavior or breaks the callers logic.

Reply:
As the command is a new one, there are no implementations to intercept `ssubscribed` calls. Delegating would simplify the default case and allow for further customizations.

Reply:
done